### DISCUSSION TITLE
Is there a way to change a record's created date?

### DISCUSSION THREAD

**Discussion by EverlyScott** [pocketbase/pocketbase#651] [Q&A] [answered]
I'm transferring our old database over to PocketBase and I was wondering if there's any way to change a record's created timestamp.
<img width="185" alt="image" src="https://user-images.githubusercontent.com/31575858/192427631-42cf88dd-586e-4af5-bf4c-bf0243cf0696.png">

**Comment by ganigeorgiev** [comment, marked as answer]
No, created and updated fields are meta fields and cannot be changed from the UI.

They can be changed if you operate directly with the database (eg. with `OnModelBefore*` hooks, using the sqlite3 cli, with db migration, etc.).

**Comment by silgon** [comment]
This would be great for testing purposes. For example, you have a frontend that tells you something about when the entry was created. To put it created like 1 month or year ago.

**Reply by ganigeorgiev** [reply]
@silgon You can define your own `date` fields that allows custom values (with the upcoming v0.23.0 the `created` and `updated` fields are no longer required and converted to `autodate` type fields, meaning that users can remove them and define their own fields when this is needed).
